<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>booking</class>
 <widget class="QMainWindow" name="booking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="StartDate_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="Source_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Source</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit_Button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Source_LE">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Destination_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="FClass_RB">
       <property name="text">
        <string>First</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SClass_RB">
       <property name="text">
        <string>Second</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Type_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Destination_LE">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>104</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="Ticket_RB">
       <property name="text">
        <string>Ticket</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Pass_RB">
       <property name="text">
        <string>Pass</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>173</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QRadioButton" name="OneMonth_RB">
       <property name="text">
        <string>One Month</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ThreeMonths_RB">
       <property name="text">
        <string>Three Months</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Class_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Class</string>
    </property>
   </widget>
   <widget class="QLabel" name="Duration_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="StartDate_LE">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>459</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
